Vector.cpp

```diff
@@ -1,8 +1,21 @@
 #include "Vector.h"
 #include <assert.h>
+#include <iostream>
+
+using namespace std;
 
 namespace Micromouse
 {
+	direction operator%(const direction &dir1, const direction &dir2)
```

Comment:
Ok it took a bit of scratch paper, but I totally get how this works and I think i can write a relatively simple doc explaining it.